run.py

```diff
@@ -799,36 +799,63 @@ def speaker_info(speaker_uuid: str, core_version: Optional[str] = None):
     @app.get(
         "/downloadable_libraries",
         response_model=List[DownloadableLibrary],
-        tags=["その他"],
+        tags=["ライブラリ管理"],
```

Comment:
タグも「音声ライブラリ管理」だと混乱が少なそうかも
```suggestion
        tags=["音声ライブラリ管理"],
```